services/users.js

```diff
@@ -41,6 +41,17 @@ const archiveUsers = async (usersData) => {
   }
 };
 
+const isUsernameLowercase = (username) => {
```

Comment:
this function will return false for usernames that contain non-ASCII characters, even if those characters are lowercase. For example, the function would return false for the username "bonjour", even though it is lowercase.